{{cookiecutter.project_slug}}/clients/mobile/react-native/package.json

```diff
@@ -83,15 +82,15 @@
     "@expo/config-plugins": "~7.2.2",
     "@expo/webpack-config": "^19.0.0",
     "@react-native-community/eslint-config": "2.0.0",
-    "@tanstack/eslint-plugin-query": "^5.14.6",
+    "@tanstack/eslint-plugin-query": "^5.17.20",
     "@types/i18n-js": "^3.8.3",
     "@types/lodash": "^4.14.185",
     "@types/react": "~18.2.14",
     "@types/react-dom": "~18.0.10",
     "@typescript-eslint/eslint-plugin": "^5.37.0",
     "@typescript-eslint/parser": "^5.37.0",
     "dotenv-cli": "^6.0.0",
-    "eslint": "^7.28.0",
+    "eslint": "^8.56.0",
```

Comment:
npm install was failing for fresh RN cookiecut projects so I fixed this with an upgrade here.

Reply:
ha I JUST ran into this bootstrapping a project. Was going to open a PR